tests/meltano/core/plugin/test_plugin_settings.py

```diff
@@ -919,12 +924,14 @@ def test_find_setting_aliases(self, tap, plugin_settings_service_factory):
         subject.set("aliased_3", "value_3")
         assert subject.get("aliased") == "value_3"
 
+    @pytest.mark.order(-1)
     def test_strict_env_var_mode_on_raises_error(self, subject):
         subject.set([FEATURE_FLAG_PREFIX, str(FeatureFlags.STRICT_ENV_VAR_MODE)], True)
         subject.set("stacked_env_var", "${NONEXISTENT_ENV_VAR}")
         with pytest.raises(EnvironmentVariableNotSetError):
             subject.get("stacked_env_var")
 
+    @pytest.mark.order(-1)
```

Comment:
Ditto duplicated order indexes

Reply:
Same as with the other duplicate, it's intentional.